statics/css/legend.css

```diff
@@ -6,7 +6,20 @@
   border-radius: 4px;
   padding: 10px;
   color: white;
-  text-align: right;
+  text-align: left;
   min-width: 200px;
   max-width: 400px;
 }
+
+.topology-legend #filter {
```

Comment:
please change these lines by

```
.topology-legend p {
  width: 300px;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
  float: left;
  clear: both;
  margin: 0;
}
```
it will do the job adding '...' at the end

No need to add a new div in the template, you just need to convert the three "span" to "p"

Reply:
ok